### PR TITLE
qt6

### PR DESCRIPTION
**Pull request by ddennedy**

### CODE REVIEW

File: org.shotcut.Shotcut.yaml [hunk 1]
```diff
@@ -283,26 +283,29 @@ modules:
...
     post-install:
       - install -d /app/extensions/Plugins
     sources:
       - type: git
         url: https://github.com/mltframework/shotcut.git
-        commit: bc48b344fb05d5d3c6a88d9f047c972f449f950f
-        tag: v22.12.21
+        #commit: bc48b344fb05d5d3c6a88d9f047c972f449f950f
+        #tag: v22.12.21
+        branch: qt6
```
Comment (hfiguiere) [inline org.shotcut.Shotcut.yaml:308]: it shouldn't be a bare branch.

**Comment by hfiguiere** [comment]
jack2 should be removed (it's in the 6.4 runtime)

the plugins version should be set to 22.08 since this is what the 6.4 runtime is based off.

**Comment by hfiguiere** [comment]
there are two places were `commmit` is spelled with 3 `m`, and that would be a good time to fix it. The commits got ignored and the tag used. But there are warnings  in the build log.

**Comment by ddennedy** [comment]
> there are two places were `commmit` is spelled with 3 `m`

Thanks Hugo!

**Comment by ddennedy** [comment]
@hfiguiere do you know anything about this error?
```
$ flatpak install --user https://dl.flathub.org/build-repo/129327/org.shotcut.Shotcut.flatpakref
error: Can't load uri https://dl.flathub.org/build-repo/129327/org.shotcut.Shotcut.flatpakref: Server returned status 403: Forbidden
```

**Comment by hfiguiere** [comment]
there have been some infra problems lately I wouldn't be suprise it's one of the downfall.

**Comment by hfiguiere** [comment]
bot, build

**Pull request closed without merging**